api/types/videotime/videotime.go

```diff
@@ -0,0 +1,103 @@
+/*
+AUTHORS
+  Scott Barnard <scott@ausocean.org>
+
+LICENSE
+  Copyright (c) 2023-2024, The OpenFish Contributors.
```

Comment:
Generally you only need to add the current year to a newly created file, a range means that the file has been created and changed over the span of the specified time.